packages/salesforcedx-vscode-apex/package.json

```diff
@@ -310,6 +310,11 @@
           "type": "boolean",
           "default": false,
           "description": "%apex_semantic_errors_description%"
+        },
+        "salesforcedx-vscode-apex.enable-completion-statistics": {
```

Comment:
@AnanyaJha - languageServer.ts looks for 'salesforcedx-vscode-apex.advanced.enable-completion-statistics'. Will this still work?
https://github.com/forcedotcom/salesforcedx-vscode/blob/797865ce7bec1e61ed488d0b16c04ef9e16782ee/packages/salesforcedx-vscode-apex/src/languageServer.ts#L47